hello-signing.md

```diff
@@ -0,0 +1,132 @@
+- Since they've never heard of Wabbit Networks, they're a little reluctant to run network software without some validations.
+- ACME Rockets has a policy to only import Docker Hub certified software, or other approved vendors.
+- Wabbit Networks works with Docker Hub to get certified, to help with their customer confidence.
+- ACME Rockets will only deploy software that's been scanned and approved by the ACME Rockets security team. They know it's been approved because all approved software has been signed by the ACME Rockets security team.
+
+## Getting Started
+- Setup a few environment variables.  
+  >Note see [Simulating a Registry DNS Name](#simulating-a-registry-dns-name) to use `registry.wabbit-networks.io`
+  ```bash
+  export PORT=5000
+  export REGISTRY=localhost:${PORT}
+  export REPO=${REGISTRY}/net-monitor
+  export IMAGE=${REPO}:v1
+  ```
+- Install [Docker Desktop](https://www.docker.com/products/docker-desktop) for local docker operations
+- Run a local instance of the [CNCF Distribution Registry][cncf-distribution]
+  ```bash
+  docker run -d -p ${PORT}:5000 ghcr.io/oras-project/registry:latest
+  ```
+- Acquire the Notation CLI  
+Notation releases can be found at: [Notation Releases][notation-releases]  
+  ```bash
+  curl -O ~/bin/notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/notation-v0.0.1-alpha.1-linux-amd64
+  curl -O ~/.docker/cli-plugins/docker-notation https://github.com/notaryproject/notation/releases/download/v0.0.1-alpha/docker-notation-v0.0.1-alpha.1-linux-amd64
+  ```
+
+## Building and Pushing
+- Build and Push the `net-monitor` software
+  ```bash
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+
+  docker push $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+  At this point, the results are empty, as there are no existing signatures
+
+## Signing a Container Image
+
+To get things started quickly, the notation cli supports generating self signed key-pairs. As you automate the signing of content, you will most likely want to create and store the private keys in a key vault. (Detailed production steps will be covered later)
+
+- Generating a Private Key  
+  The following will generate an X-509 private key under the `./notary/` directory
+  ```bash
+  notation cert generate "wabbit-networks.io"
+  ```
+- Sign the `net-monitor:v1` container image
+  ```bash
+  notation sign $IMAGE
+  ```
+- List the image, and any associated signatures
+  ```bash
+  notation list $IMAGE
+  ```
+## Verify a Container Image Using Notation Signatures
+
+To avoid a Trojan Horse attack, before pulling an artifact into an environment, it is important to verify the integrity of the artifact based on the identity of an entity you trust. Notation uses a set of configured public keys to verify the content. The `notation cert generate` command automatically added the public key.
```

Comment:
```suggestion
To avoid a Trojan Horse attack, and before pulling an artifact into an environment, it is important to verify the integrity of the artifact based on the identity of an entity you trust. Notation uses a set of configured public keys to verify the content. The `notation cert generate` command automatically added the public key.
```

The "The `notation cert generate` command automatically added the public key." line feels awkward -- should it be "adds" in this case? Also, I'm not sure where it's added to, or when in this process this addition happens.

Reply:
Adding certificate to as trusted for verification should probably be an explicit step, otherwise any test certificates created by user will be automatically added into the verification policy, and be unintentionally used in production if they copied over the same policy.

Reply:
See #86